source/Dictionary.h

```diff
@@ -32,6 +32,9 @@ class Dictionary : private std::vector<std::pair<const char *, double>> {
 	double Get(const char *key) const;
 	double Get(const std::string &key) const;
 	
+	// Compare two dictionaries to see if they contain exactly the same data.
+	bool Equals(Dictionary &other);
```

Comment:
```suggestion
	bool Equals(const Dictionary &other) const noexcept;
```

Reply:
Yes, should be const. I'll update this in the next commit.